TC_RELEASE.md

```diff
@@ -0,0 +1,52 @@
+# Release Terra-Components
```

Comment:
Mit oder ohne Bindestrich? In der DoD war es ohne geschrieben.

Reply:
Würde "mit" sagen.. was sagt @dantrau ?

Reply:
ja, in der DoD hatte ich es entfernt

Reply:
Okay ich pass es an 😄